### ISSUE TITLE
Draft messages get not handled correctly (seems Encoding/Umlaut issues)

### ISSUE THREAD

**Issue by mstenz** [hmailserver/hmailserver#293]
This is a reference to #266
I have now a debug log available, but because of GDPR issues i cannot publish this here. 
Please send me an email to mstenz-design@web.de if you are interested in the debug log and i can send it to you for your personal use only. 

_Originally posted by @martinknafve in https://github.com/hmailserver/hmailserver/issues/266#issuecomment-522034572_

**Comment by mstenz** [comment]
The test email i have sent was not delivered correctly to the recipient and was completely empty. This behaviouir is new to me as i always have sent emails to external participants. 
Here is the email as it was received from the receiver:

> Return-Path: <info@laminierungsprofi.de>
Received: from cowmedia.de ([91.205.172.53]) by mx-ha.web.de (mxweb110
 [212.227.17.8]) with ESMTPS (Nemesis) id 1N2D5Q-1iPQ5P3lx0-013fq2 for
 <mstenz-design@web.de>; Fri, 16 Aug 2019 17:06:10 +0200
Received: from msclient (p57BA6A95.dip0.t-ipconnect.de [87.186.106.149])
	by cowmedia.de with ESMTPSA
	(version=TLSv1.2 cipher=ECDHE-RSA-AES256-SHA384 bits=256)
	; Fri, 16 Aug 2019 17:06:04 +0200
Message-ID: <BEAF4A3C-49AB-4401-90A8-D0D27DFBFEBF@cowmedia.de>
Envelope-To: <mstenz-design@web.de>
X-UI-Filterresults: unknown:5;V03:K0:5s9WKz6yaLY=:IF01+3yPW/Z4JzTy65mSy25GBX
 MHrYplKnzeb7OK/g9jGffvwCjxUD/Is0LLnRGcVgfVsQ1yvqSu0zl7WeMzwQ4xqwMW0HsLoEG
 JgXuMS29Jk+eadd2zYsHNqLaX0oysMRJsHNu1PdxJKZV9cCcYvlIkZebe1Ya5vL40WYajLggK
 7OxTRjK4bDuRqANUQnz//71GIvxh/d0InLt2WpPnyett+VCdV7bR1dtxIGsBaLu0RtILRIyeB
 vecLo2ntW6PHYGEE3r2y7pIjRq3+tFcYOoCGYNyf3TH/xoL/I6B+W+6f6cZLzUepU1EUgpRCq
 /hDz4GEXmdeiN7QzUPApgnGs2YngAgOg9GoQSS+eSd2za5i5CZVV+u48UhrmRPoVyUiHNe8gE
 2crvKXvLOT5KR+BYycbuA83aa+hPMzLkydRcbZ0OjsGEXhCUq3+eQ/vje+6VZ8y9XrCC7eKVG
 g/Kneeql8AkjoyTxascZPfb3VwiwHtUYr1YAo9QP2yWnP6TBtnT6xBqLFeH3S/TDOgcP4ZZ9W
 fk9Qxuk5BwIwWc3bAF9Y1vYfbYhz6dX2ZJABfn0Se25/7QS3Owli7DQraB0Eml2kYtdVJ8yfw
 lZQ7frdzJasctISAhckpBHDUCORObV7RNsums+QFo/TrwIg5/nJiS6PfbgvOUwOBAkXj6rbsZ
 MEuy//R1PaUfPTQL3ehemX5Ubb5/QagWWgoriULTUbQoXb4zQ3en3LXYHA4LRcVS1zc2oBoG9
 Ji9WmgNpcKQmZEZBOLOfvFz/77Au1y2X0zbs5dvW9oZoP2FKLBT7Oe3TzEWje1MtCv9gufsna
 Vk+Rj/OBAkJEZxPBEXC8ZVicY+//U//dyBvcimLsMg/SqwLaCJlvfSjy9/FdO3EMGsppCPgfv
 /qyHMD0CK8eWlLegtlK6QkJTSGfqq2vjlqYnhTbu35AR7ZTSRD4yDBhXJjfN9xSCL83lnM1rV
 TR0RPeAPVdMHH4wfzU9qGOzOSq+6ey/Wv/6zOck1dx52szTCwYrJXBkmcvspNKRrauQDpIWP+
 t+aMTHa0Yu/meKKdGBXaMVDfq8WZyXVV4xRdjzKTGPCXGgO7YQt4XdortHXpJwa/D36oCS5m/
 DZA/rRLjEXrrWTiyDYBKtCj2ocIrR02/d2uLOvEXzDIBlduplNiGXXQrfuFGdqhmB/bonmdrc
 NWKFgwJ3+icdKl3CFgqq3fxtzogrHyEnXKpW14Iu/uRKrbzyws6J7/Uo3zhmyrn9fz9s11J97
 qw0fjYvsaL2Cq1gR8aSltaGk9xOJ1hobL6QCnD+2btsgr+gGJNCkyOXT3s9YOjW29hvoOo07t
 ut5tFrH0YF4ZUO0BCTlYOFY/Nus3OXJuKJgVJVOWs1Fvqp8ShPxQVO23RTlrEQ+G4bLLBd66f
 MFYQ6Zc0CF4Y1rdsKpxzKTFcptjs40/mSByq0MF7xCmgC2bOTweGjb0CgwK23OyQ2T3HK8an4
 Xwz5HpTC8btCWaP/rmGqlxWIq0KV33hXxjAeSJrzN75sJykpKh86YgHHXGcbriHnDJQQOdeC9
 xVVtkc9XTgfS0hLkETHDG9J3XUyhpbqTI7+o1bmzaigrj4jouOigoJVZ/zjCiXey9Ix58UPDc
 Crzc4GuuN/n+ijldVTvllBVe5NmZFdu24mBjp3/QoEd7wW+GMx6g8yAaXDg7hYHfELTHg8GLo
 8kZ4wBN2PXxmwIHhBbHREgHIMN7DsWwLpaj9rhKGGGJpsVybHgQyXi41GrkYAX8ue5oESBc2d
 HjdhVxusKrf+a3qcka24zLmchFrCn0M1wBDATfzxfVAZVlNGa0NsSAveUS3J9PkCnofACfyqY
 lJ+9zS/M4vZ0hx/ZHCH8uyS42DIB52UuoQHcg99l4EY8RX7rlciXrhfHaDmIVkvH6/bgBEc4t
 JcAVvT7dvxovfE+jTTVSzAyRAaBfBPYwsmvDHdxX4rN9Z4b8snhnNhLJ/L0NQiQES9K/jnnmc
 GuaU9QwNGxn7eeiV2iQviYdMGKq6emwHKmnxOQ9HYTbmtAvVveJpUhHkWLzWzKBDrSgphNWfU
 R07iGpZ6eMX1sGqx04AC3S+GsYqZ9IHVR7c/q/WOQynUIhXmuO+nqSnPso4FbwbX2Y51O/YYR
 s2lfqzb1xh+A4xIAP0bsCuQPJwTkpEOtzdNvFvq0U91cKBoFwYmHhwK7b1/FhNRezfdOZl+z2
 ukAIY7cZsb7bTAGWnM+QoN2OW0HNLEqOb61PiHofUeZHQ7WDi7eTYMwYBz1I2ADXoxsSgmzV/
 rHMzWg2dTSonepHpr9dRZiHMW20hCyM4T2HFcwv6GxAM7CERwzBltTYCMit5Cc252RXJA858T
 cCjv/w

As you can see subject and content of the email is completely missing.

**Comment by martinknafve** [comment]
Please send the logs to me at martin@hmailserver.com.

**Comment by mstenz** [comment]
Thank you. I have sent you the log via email.

**Comment by mstenz** [comment]
Any Idea what it can be?

**Comment by mstenz** [comment]
Hi Martin, have not heard back. the issue is still present and because it lead to data loss very critical. emails are not sent even they seem to be and there is no message about this at all

**Comment by victormss** [comment]
This problem also happens to me. But I have only seen it happen with Outlook, with roundcube and thunderbird it does not happen.
We use 5.6.7-B2425.

**Comment by mstenz** [comment]
@victormss Yes, it definitive does not work with Outlook, as this is the one i am using also, but i have also seen this issue with Thunderbird already on one of my older test machines.

**Comment by mstenz** [comment]
maybe you have time again to look to this issue? Outlook is the most used Email client in the world and I think fixing this very annoying issue that come in coherance with data loss for the writer of an email should be important enough to have a look on.

**Comment by SaculRennorb** [comment]
I'm not completely certain that I am having the same issue, but I have been using hMailserver for a bit now and noticed an at least similar problem to the one described.

As far as I can tell this was originally about a potential localization issues, which might also be present in my setup:
The mail server (v 5.6.9-B2607) is running on Windows Server 2016 (German as the primary language), while the client is Thunderbird 115.13.0 (64-bit) on a Win10 machine with English as the primary language.
The mail user has a umlaut in its name, and messages do not get removed from the draft folder. in fact multiple versions of the same mail draft pile up while writing emails, which I presume is in fact one and the same issue.
I have been using Thunderbird for a long time and with loads of clients in parallel, and none of the other ~10 or so accounts on different servers experience similar issues. Gmail, IONOS hosting and docker mail server all handle drafts as expected, so I'm fairly certain its an issue with hMailserver specifically.

Normal sending and receiving works perfectly fine as far as I can tell. I am also able to manually delete emails.

I'm happy to supply more information if there is something specific you need to look into this.

**Comment by OutbackMatt** [comment]
I suspect that this is a different issue.
We should really be having this conversation on the user forum please.
https://www.hmailserver.com/forum/
